hs_model_program/templates/pages/modelprogramresource.html

```diff
@@ -6,105 +6,101 @@
-                            {% for name in mpmetadata.get_documentation_list %}
-                                <tr>
-                                    {% if forloop.counter0 > 0 %}
-                                        <th></th>
-                                    {% else %}
-                                        <th class="text-muted">Documentation</th>
-                                    {% endif %}
-                                    <td><a id="permalink"
-                                           href="http://{{ request.get_host }}/hsapi/resource/{{ cm.short_id }}/files/{{ name }}"
-                                           target="_blank">{{ name }}</a></td>
-                                </tr>
-                            {% endfor %}
-                        {% endif %}
-
-
-                        {% if mpmetadata.get_releasenotes_list|first %}
-                            {% for name in mpmetadata.get_releasenotes_list %}
-                                <tr>
-                                    {% if forloop.counter0 > 0 %}
-                                        <th></th>
-                                    {% else %}
-                                        <th class="text-muted">Release Notes</th>
-                                    {% endif %}
-                                    <td>
-                                        <a id="permalink"
-                                           href="http://{{ request.get_host }}/hsapi/resource/{{ cm.short_id }}/files/{{ name }}"
-                                           target="_blank">{{ name }}</a>
-                                    </td>
-                                </tr>
-                            {% endfor %}
-                        {% endif %}
-                        </tbody>
-                    </table>
-                    <hr style="border:0">
-
+        <div class="col-md-12">
+            {# Add the view mode html code below #}
+            {% if mpmetadata %}
+                {########################}
+                {#       CONTENT FILES     #}
+                {########################}
+                {% if mpmetadata.get_engine_list|first or mpmetadata.get_software_list|first or mpmetadata.get_releasenotes_list|first  or mpmetadata.get_documentation_list|first  %}
+                <legend>Content files</legend>
+                <table class="custom-table">
+                    <tbody>
+                    {% if mpmetadata.get_engine_list|first %}
+                        {% for name in mpmetadata.get_engine_list %}
+                            <tr>
+                                {% if forloop.counter0 > 0 %}
+                                    <th></th>
+                                {% else %}
+                                    <th class="text-muted">Computational Engine</th>
+                                {% endif %}
+                                <td><a id="permalink"
```

Comment:
Suggest a different id.

Reply:
@engineSound There are a bunch of tags with this id. I suggest we get rid of these attributes instead since they are not being referenced anywhere in the project that I am aware.